### DISCUSSION TITLE
Post Processing Bloom 효과의 적용 관련 문의

### DISCUSSION THREAD

**Discussion by hhnmsLeesan** [naverz/zepeto-studio-kor#765] [5. Q&A]
안녕하세요!

현재 Color Grading 등의 다른 Post Processing 효과 등은 정상 적용되고 있는데 Bloom효과는 에디터와 동일하게 적용되고 있지 않는 현상을 발견하였습니다. 
모든 상태의 게임오브젝트(static이건 아니건)/ 모든 종류의 머티리얼(baked emissive이건 아니건)에 적용이 되지 않고 있고 저희가 작업중인 다른 프로젝트에서도 이 같은 현상이 발생하는 것으로 보아 Bloom이 제페토 모바일 앱에서는 적용되지 않는 것으로 추측하고 있습니다.

이 부분과 관련하여 저희의 착오가 있는 건지, 실제로 그런 것인지 여쭙고자 질문 글을 남겨 봅니다.
감사합니다!

**Comment by x4cf** [comment]
크리에이터 참이슬님의 월드 "Studio Ci Group World" 를 보면 포스트프로세싱의 Bloom 기능은 잘 작동하는것으로 보입니다 :)
